app/kubemci/pkg/gcp/forwardingrule/interfaces.go

```diff
@@ -28,4 +28,6 @@ type ForwardingRuleSyncerInterface interface {
 	DeleteForwardingRules() error
 	// GetLoadBalancerStatus returns the struct describing the status of the given load balancer.
 	GetLoadBalancerStatus(lbName string) (*status.LoadBalancerStatus, error)
+	// GetForwardingRules returns all forwarding rules used by MCI.
+	GetForwardingRules() ([]status.LoadBalancerStatus, error)
```

Comment:
How about calling this ListLoadBalancerStatuses?
From GetForwardingRules, I expect it to return compute.ForwardingRule.